index.html

```diff
@@ -7,15 +7,77 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <nav class="navbar"> 
+            <img class="logo" src="img\karma-logo.svg" alt="karma-logo">
+                
+                    
+                <a class="icon"><img src="img\menu-hamburger.svg" alt="Menu icon"> </a>
+                
+                    <div class="right">
+                            <a href="#" class="current-tab">Meet Karma</a>
+                            <a href="#">How it works</a>
+                            <a href="#">Store</a>
+                            <a href="#">Blog</a>
+                            <a href="#">Help</a>
+                            <a href="#">Login</a>
+                    </div>
+        </nav>
+        
+    </header>
+    <section class="intro">
+        <div class="content">
+            <h1>Introducing Karma</h1>
+            <h3>Bring WIFI with you, everywhere you go</h3>
+            <div>
+                <button type="button" class="button">Learn More</button>
+            </div>
+        </div>
+    </section>
+    <div class="statement">
+        <h1>Everyone needs a little Karma.</h1>
+    </div>
```

Comment:
This `h1` should be inside the next section (and the `div` that wraps it is not necessary) as its the heading for the 3 large orange icons.

Reply:
Resolved